docs/dev/dev_contributing.md

```diff
@@ -13,12 +13,13 @@ Documentation is built using [mkdocs](https://www.mkdocs.org/). The [Docker base
 
 The branching policy includes the following tenets:
 
-- The develop branch is the branch of the next major and minor paired version planned.
+- The develop branch is the branch for bug fixes.
+- The next branch is the branch the next major and minor paired version planned.
```

Comment:
```suggestion
- The next branch is the branch for new features. It is therefore the branch from which the next planned `stable-<major>.<minor>` version will be sourced.
```

Reply:
I don't read this as accurate `It is therefore the branch from which the next planned `stable-<major>.<minor>` version will be sourced.`